test/test262-parser-tests/unsupported-features.txt

```diff
@@ -0,0 +1,9 @@
+explicit-resource-management
+regexp-modifiers
+import-assertions
+import-attributes
+regexp-duplicate-named-groups
+regexp-v-flag
+arbitrary-module-namespace-names
+source-phase-imports
+source-phase-imports-module-source
```

Comment:
@3cp Can you confirm this is the correct list of features that we don't support?

Reply:
Regexp modifiers should work.
Meriyah doesn't parse regexp internal, the whole regex is one tokenValue, parsed by native RegExp provided by the runtime (which runs meriyah), `try { return new RegExp(...) } catch ...`.

In same sense, regexp-duplicate-named-groups properly also works.

Reply:
class-static-block is not supported.

Reply:
class-static-methods-private is not supported.

Reply:
> Meriyah doesn't parse regexp internal

I didn't know that at first, I think it's really bad to rely on runtime behavior, maybe we should do something in future.